libmozevent/utils.py

```diff
@@ -146,6 +146,12 @@ class AsyncRedis(object):
     async def connect():
         if AsyncRedis.redis.get(None) is None:
             AsyncRedis.redis.set(
-                await aioredis.from_url(os.environ["REDIS_URL"], decode_responses=False)
+                await aioredis.from_url(
+                    os.getenv("REDIS_TLS_URL", None)
+                    if os.getenv("REDIS_SSL_URL", None) is not None
+                    else os.getenv("REDIS_URL", None),
+                    decode_responses=False,
+                    ssl_cert_reqs=None,
+                )
```

Comment:
It seems that this caused an error:

https://community-tc.services.mozilla.com/tasks/B00xAklaTmGPfAb4pD-A1Q/runs/0/logs/public/logs/live.log#L282